_chapters/chapter_02.md

```diff
@@ -48,7 +48,7 @@ layout: rails_tutorial
 
 {% steps %}
 {% list %}
-  1.  Open the `bookstore` directory in your text editor and open `db/migrate/YOUR_TIMESTAMP_create_books.rb`.
+  1.  Open the `bookstore` directory in your text editor and look at `db/migrate/YOUR_TIMESTAMP_create_books.rb`.
```

Comment:
I wonder if we should change this to "Open `db/migrate/YOUR_TIMESTAMP_create_books.rb` in your text editor." What we have now is a little wordy.